content/en/docs/setup/additional-setup/cni/index.md

```diff
@@ -91,47 +107,71 @@ This race condition is mitigated by a "detect and repair" method.
 Please take a look at [race condition & mitigation](#race-condition--mitigation) section to understand the implication of this mitigation.
 {{< /tip >}}
 
+### Installing with Helm
+
+The Istio CNI and Istio discovery chart use different values that require you set the following either in an overrides values file or at your command prompt when installing the `istiod` chart.
+
+* `values.istio_cni.enabled` should be set to the same value as `values.cni.enabled` to manage network annotations when chaining CNI plugins.
+* `values.istio_cni.chained` should be set to the same value as `values.cni.chained` to manage network annotations when chaining CNI plugins.
+
+{{< text bash >}}
+$  helm install istiod istio/istiod -n istio-system --set values.istio_cni.enabled=true --wait
+{{< /text >}}
+
 ### Hosted Kubernetes settings
 
 The `istio-cni` plugin is expected to work with any hosted Kubernetes version using CNI plugins.
 The default installation configuration works with most platforms.
 Some platforms required special installation settings.
 
-* Google Kubernetes Engine
-
-    {{< text yaml >}}
-    apiVersion: install.istio.io/v1alpha1
-    kind: IstioOperator
-    spec:
-      components:
-        cni:
-          enabled: true
-          namespace: kube-system
-      values:
-        cni:
-          cniBinDir: /home/kubernetes/bin
-    {{< /text >}}
-
-* Red Hat OpenShift 4.2+
-
-    {{< text yaml >}}
-    apiVersion: install.istio.io/v1alpha1
-    kind: IstioOperator
-    spec:
-      components:
-        cni:
-          enabled: true
-          namespace: kube-system
-      values:
-        sidecarInjectorWebhook:
-          injectedAnnotations:
-            k8s.v1.cni.cncf.io/networks: istio-cni
-        cni:
-          cniBinDir: /var/lib/cni/bin
-          cniConfDir: /etc/cni/multus/net.d
-          cniConfFileName: istio-cni.conf
-          chained: false
-    {{< /text >}}
+{{< tabset category-name="cni-platform" >}}
+
+{{< tab name="Google Kubernetes Engine" category-value="gke" >}}
+
+{{< text yaml >}}
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
```

Comment:
maybe we just do 

{{< text bash >}}
$ istioctl install -set profile=openshift
{{< /text >}}